homework-15/letter.md

```diff
@@ -0,0 +1,5 @@
+Hello Alina! I have been on a trip to Minsk and in this letter, I tell you about it.
+When I was at the conference, named "Beer Js Summit" my Belorussian friend, Irina, invited me on a trip in Nesvizh and Mir. When I backed to Moscow and quit, I thought about my free time and decided back to Minsk through two weeks.
+I arrived in Minsk on Friday and on Saturday we went to Nesvizh by train. Nesvizh is a city with a castle saved from the sixteenth century. We visited the town hall, old gate to Slutsk and castle. It was interesting. I felt myself in Europe because of architecture. I liked the town pointers, like a dick, history about Randzevill family and theatre. After that, we sat on a train and went to Minsk.
```

Comment:
I felt as I was in Europe
because of **the** 
dick-like-looking pointers
the theatre